src/agentscope/pipelines/functional.py

```diff
@@ -40,123 +41,133 @@ def sequentialpipeline(
     return msg
 
 
+def _operators(operators: Operators, x: Optional[dict] = None) -> dict:
+    """Syntactic sugar for executing a single operator or a sequence of
+    operators."""
+    if isinstance(operators, Sequence):
+        return sequentialpipeline(operators, x)
+    else:
+        return operators(x)
+
+
 def ifelsepipeline(
-    x: dict,
     condition_func: Callable,
-    if_body_operator: _Operator,
-    else_body_operator: _Operator = placeholder,
+    if_body_operators: Operators,
```

Comment:
Do we need to add Sequence[Operators] to the type here?

Reply:
`Operators` is an alias for `Union[Operator, Sequence[Operator]]`